spec/features/staff/organisation_show_page_spec.rb

```diff
@@ -46,7 +48,7 @@
         end
 
         within("##{fund.id}") do
-          expect(page).to_not have_content I18n.t("summary.label.activity.publish_to_iati.yes")
+          expect(page).to_not have_content I18n.t("summary.label.activity.publish_to_iati.true")
```

Comment:
This doesn't seem to belong here? Also, begs the question of whether we should raise on missing translation in tests and whether it's ok to have a generic I18n key for these  booleans? 🐰 🕳️

Reply:
You're right this was me tidying up and I should put it in another commit 🤦‍♀️